sources/packages/backend/workflow/BPMN/camunda-8/parttime-assessment-2021-2022.bpmn

```diff
@@ -673,6 +657,136 @@
+    <bpmn:sequenceFlow id="Flow_0qgamw3" sourceRef="Gateway_02qbvrn" targetRef="Gateway_0pzv80g" />
+    <bpmn:sequenceFlow id="Flow_1v46bg2" name="Course Load Less than 35" sourceRef="Gateway_0pzv80g" targetRef="Event_10tndoy">
+      <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">=offeringCourseLoad &lt;= 34</bpmn:conditionExpression>
+    </bpmn:sequenceFlow>
+    <bpmn:sequenceFlow id="Flow_1lq8ab4" name="Course Load 35 And Up" sourceRef="Gateway_0pzv80g" targetRef="Event_0wg31nu">
+      <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">=offeringCourseLoad &gt; 34</bpmn:conditionExpression>
+    </bpmn:sequenceFlow>
+    <bpmn:sequenceFlow id="Flow_00mw736" sourceRef="Event_10tndoy" targetRef="Gateway_0rrzb1g" />
+    <bpmn:sequenceFlow id="Flow_1hlncla" sourceRef="Event_0wg31nu" targetRef="Gateway_0rrzb1g" />
+    <bpmn:sequenceFlow id="Flow_0lp9mvw" sourceRef="Gateway_0rrzb1g" targetRef="Event_0re3zsg" />
+    <bpmn:sequenceFlow id="Flow_0temot0" sourceRef="Event_0re3zsg" targetRef="Gateway_1m98y7x" />
+    <bpmn:sequenceFlow id="Flow_0djv5vn" name="Offering weeks greater than 17 and less than or equals 34" sourceRef="Gateway_1m98y7x" targetRef="Event_12kp8ra">
+      <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">=offeringWeeks &gt; 17 and offeringWeeks &lt;= 34</bpmn:conditionExpression>
+    </bpmn:sequenceFlow>
+    <bpmn:sequenceFlow id="Flow_18af5gw" name="Offering weeks greater than 34" sourceRef="Gateway_1m98y7x" targetRef="Event_0sb0723">
+      <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">=offeringWeeks &gt; 34</bpmn:conditionExpression>
+    </bpmn:sequenceFlow>
+    <bpmn:sequenceFlow id="Flow_1v61q2r" name="Offering weeks less than or equals 17" sourceRef="Gateway_1m98y7x" targetRef="Event_01fzlt0">
+      <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">=offeringWeeks &lt;= 17</bpmn:conditionExpression>
+    </bpmn:sequenceFlow>
+    <bpmn:sequenceFlow id="Flow_16xshp2" sourceRef="Event_12kp8ra" targetRef="Gateway_0r706cf" />
+    <bpmn:sequenceFlow id="Flow_0iqqipy" sourceRef="Event_0sb0723" targetRef="Gateway_0r706cf" />
+    <bpmn:sequenceFlow id="Flow_19bk6l4" sourceRef="Event_01fzlt0" targetRef="Gateway_0r706cf" />
+    <bpmn:sequenceFlow id="Flow_0jaycgo" sourceRef="Gateway_0r706cf" targetRef="Event_0x1vhw9" />
+    <bpmn:sequenceFlow id="Flow_0jntyac" sourceRef="Event_1j6ep9h" targetRef="Activity_0qzkp53" />
+    <bpmn:sequenceFlow id="Flow_114jf7v" sourceRef="Event_0x1vhw9" targetRef="Event_1j6ep9h" />
+    <bpmn:sequenceFlow id="Flow_0201yrq" name="Additional Transportation is 0km" sourceRef="Gateway_1o496q2" targetRef="Event_15d88qb">
+      <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">=calculatedDataAdditionalTransportKm = 0</bpmn:conditionExpression>
+    </bpmn:sequenceFlow>
+    <bpmn:sequenceFlow id="Flow_04xv6y2" name="Additional Transportation 280km And Less" sourceRef="Gateway_1o496q2" targetRef="Event_1lqekii">
```

Comment:
Could we have a const for `additionalTransportationOnsiteDistance` and refer to that name instead of having a hardcoded number?

Reply:
It is just used in a condition in 2 decisions and we haven't created constants for such decision parameters in workflow as they are not used in other places. 

IMO I feel there is no great benefit on constant in these cases.

Reply:
After talking to Lewis I changed my mind and don't think we should change it. Resolving.